examples/nemo/nemo.yaml

```diff
@@ -0,0 +1,59 @@
+# Distributed training a BERT model with Nvidia NeMo
+#
+# Finetunes a BERT-like model on the GLUE CoLA task. Uses the NeMo toolkit
+# to train across multiple nodes, each node having a V100 GPU.
+#
+# Usage:
+#   sky launch -c nemo nemo.yaml
+#   sky down nemo
+
+resources:
+  accelerators: V100:1
+
+num_nodes: 2
+
+setup: |
+  conda activate nemo
+  if [ $? -eq 0 ]; then
+      echo "conda env exists"
+  else
+      conda create -y --name nemo python==3.10.12
+      conda activate nemo
+  
+      # Install PyTorch
+      pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118
+      
+      # Install nemo
+      sudo apt-get update
+      sudo apt-get install -y libsndfile1 ffmpeg
```

Comment:
Seems we installs `ffmpeg`, but is training on language tasks. Should we train on some CV tasks instead?

Reply:
This is also from the [official install instructions](https://github.com/NVIDIA/NeMo#pip), I thought of leaving it in if people use NeMo for multi-modal tasks. Let me know if you want to remove it.

Reply:
I would prefer to remove these to keep our setup minimal. I think we can have another example yaml for multi-modal task that include these commands. Wdyt?